pkg/tsdb/cloudmonitoring/time_series_filter.go

```diff
@@ -309,7 +309,7 @@ func (timeSeriesFilter *cloudMonitoringTimeSeriesFilter) parseToAnnotations(dr *
 			annotation["time"] = append(annotation["time"], point.Interval.EndTime.UTC().Format(time.RFC3339))
 			annotation["title"] = append(annotation["title"], formatAnnotationText(title, value, series.Metric.Type,
 				series.Metric.Labels, series.Resource.Labels))
-			annotation["tags"] = append(annotation["tags"], tags)
+			annotation["tags"] = append(annotation["tags"], "")
```

Comment:
are the `tags` required here? what happens if you remove this line?

Reply:
I believe they are. The framework expects tags to be returned, even though they are empty